deploy/openshift/clusterrole.yaml

```diff
@@ -22,6 +22,7 @@ rules:
   - statefulsets
   - replicasets
   - replicationcontrollers
+  - routes
```

Comment:
We need to notify Maistra and upstream Istio with this new change. Create Maistra JIRA and upstream istio issues for these.

Reply:
My bad - this isn't related to kubernetes changes, so no upstream istio notification required